tests/Maker/MakeCrudTest.php

```diff
@@ -36,11 +37,30 @@ public function getTestDetails()
             ->setRequiredPhpVersion(70200),
         ];
 
+        yield 'crud_basic_custom_controller' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeCrud::class),
+            [
+                // entity class name
+                'SweetFood',
+                'SweetFoodAdminController', // default controller
+            ])
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeCrudCustomController')
```

Comment:
Fixtures are missing in the PR. Is it possible to use the `MakeCrud` fixtures?

Reply:
bah, thanks! I need a new directory so that the generated test inside can go to the new URL, which is now `/sweet/food/admin`